src/dags/common/__init__.py

```diff
@@ -27,10 +30,47 @@
 SHARED_DIR = env("SHARED_DIR", "/tmp")
 
 
+class MonkeyPatchedSlackWebhookHook(SlackWebhookHook):
```

Comment:
Het toevoegen van extra logica aan run() de SlackWebhookHook lijkt mij een noodzakelijk tijdelijke oplossing. 
Goed dat je ook in de docstring aangeeft waarom dit gedaan wordt.